### ISSUE TITLE
Changes to checkout.liquid are not being pushed up.

### ISSUE THREAD

**Issue by joshuaiz** [internalfx/quickshot#75]
Hello, 

Still loving Quickshot.

I'm using it with a Shopify Plus client and it seems that any changes to `checkout.liquid` are not being pushed up to the target store. All other files are fine including `theme.liquid`.

Is there anything that could be blocking changes to `checkout.liquid` or otherwise preventing these changes to be pushed up?

My best guess is it is some sort of permissions issue.

Here are the steps to reproduce:

1. Create target
2. Use `theme watch --sync` command
3. Make local changes to `checkout.liquid`
4. Save file; Terminal reports: 02:48:25 pm - Added/Updated theme/layout/checkout.liquid
5. Reload Checkout page and/or check in Themes > [theme] > Edit theme code
6. `checkout.liquid` file is not updated.

If you need more information, let me know. Thanks!

**Comment by internalfx** [comment]
I'm looking at the code right now and I can't see any reason it wouldn't work

**Comment by internalfx** [comment]
Can you try using `theme upload` directly?

Let me know...

**Comment by joshuaiz** [comment]
I just tried using this:

`$ qs theme upload theme/layout/checkout.liquid` 

and it started to upload every file in the theme(!!!!) What am I missing here?

I only have 1 target configured.

**Comment by internalfx** [comment]
You want to use the `--filter` option

**Comment by joshuaiz** [comment]
@internalfx thanks.

Here's what I tried:

```bash
$ qs theme upload --filter checkout.liquid
11:04:28 pm - Uploaded 0 files.
$ qs theme upload --filter theme/layout/checkout.liquid
11:05:08 pm - Uploaded 0 files.
$ qs theme upload --filter /theme/layout/checkout.liquid
11:05:37 pm - Uploaded 0 files.
$ qs theme upload theme/layout/checkout.liquid --filter 
11:07:26 pm - Uploaded 0 files.
```

Doesn't seem like it's uploading or I'm doing something wrong.

**Comment by internalfx** [comment]
`qs theme upload --filter checkout.liquid` should have worked....

**Comment by joshuaiz** [comment]
@internalfx yep it should have but it doesn't as you can see above: 'Uploaded 0 files.' 

My best guess is it is a permissions issue since `checkout.liquid` is only available to Shopify Plus stores so maybe there are different permissions for that?

I looked at my private app settings and couldn't see anything that jumped out at me. I can follow up also with Shopify Plus support.

**Comment by internalfx** [comment]
I unfortunately don't have a Shopify plus account I can test this with.

Please do let me know what you find out. 

For reference the API call I'm using is this.

https://shopify.dev/docs/admin-api/rest/reference/online-store/asset?api[version]=2020-04#update-2020-04
![image](https://user-images.githubusercontent.com/583701/81341947-457be480-9067-11ea-90d7-7f84a8fd930e.png)

**Comment by joshuaiz** [comment]
Sorry for the delay on this - I contact Shopify Plus support who elevated the issue to their dev team and this is what they just came back with:

----
I've chatted with our partner support team about the issues that you are having with Quickshot not publishing updates to your checkout.liquid. At this time, they have advised that it would be best if the developer of the app reached out directly through their partner dashboard for more information on this.
----

If you don't already have a Shopify partners account, you can create one here: https://www.shopify.com/partners

Hopefully they can give you some more information directly.

I can confirm that this is still an issue so any changes to `checkout.liquid` have to be made in the Shopify admin and don't get pushed up via Quickshot. Not an emergency by any means but I would assume other people are having the same issue.

**Comment by internalfx** [comment]
@joshuaiz has anything changed regarding this issue?

**Comment by joshuaiz** [comment]
This issue still remains - pushes to `checkout.liquid` do not work with Quickshot.

**Comment by internalfx** [comment]
@joshuaiz ok, thank you, I'll take a deeper look.
